weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/CallsTable.tsx

```diff
@@ -522,6 +522,10 @@ export const CallsTable: FC<{
       : [];
   }, [allRowKeys, columnVisibilityModel, tableData]);
 
+  const refColumnsToExpand = useMemo(() => {
+    return Array.from(expandedRefCols).filter(col => columnsWithRefs.has(col));
```

Comment:
under what conditions does `expandedRefColumns` contain cols not in `columnsWithRefs`?

Reply:
Many actually, although I did not bother to figure out why. Example: 

When I first load the page:
<img width="400" alt="Screenshot 2024-08-29 at 4 46 08 PM" src="https://github.com/user-attachments/assets/788418d3-ac2b-459a-9e1b-44269c084e50">

When I expand "response":
<img width="401" alt="Screenshot 2024-08-29 at 4 46 31 PM" src="https://github.com/user-attachments/assets/ac0b56ed-9db6-4ce0-9fc4-2378e942e391">

Reply:
oh - that is because there is some silly thing we did that tries to default to including inputs.example

```
const [expandedRefCols, setExpandedRefCols] = useState<Set<string>>(
    new Set<string>().add('inputs.example')
  );
```

Reply:
Right, made it simpler to account for that